src/string.cr

```diff
@@ -568,7 +568,7 @@ class String
         next
       end
 
-      last_is_underscore = false
+      last_is_underscore : Bool = false
```

Comment:
This change is nonsense.

Reply:
I have searched & replaced then push before going to week-end without reviewing... oops :)